notebooks/01-introduction/01-clickstream/01-introduction-clickstream.md

```diff
@@ -0,0 +1,103 @@
+# Introduction to clickstream
+
+## Background
+
+IT teams in government and commercial organizations began to recognise the value of log data generated by website-hosting technologies like [Apache HTTP Server](https://httpd.apache.org/) and [Microsoft Internet Information Services](https://www.iis.net) some time ago.
+
+For many years, this server log data remained the sole purvue of IT teams wanting to meet OLAs and SLAs, and to prevent and investigate security incidents.
+
+As the internet became critical to business, data like this became valuable for measuring business success - things that have before been used to measure the success of physical channels:
+
+* What kind of people are landing on our product page?
+* Where are most of our visitors from?
+* What are people's usage habits?
+* Do we have return visitors?
+* How long does it take for someone to buy our services?
+* What questions are people asking about our products when they arrive?
+
+As website visitors, we are all being coaxed toward achieving a particular goal that the owning organization has in mind for us, whether that's placing an order for a product, signing up for a service, or passing on contact information for sales leads.
+
+A _conversion_ is when a visitor achieves one of these goals - and each conversion is assigned a _conversion value_ depending on how important achieving that goal is. Behind that conversion is a _journey_ - understanding and tweaking that journey is a critical reason for doing analytics on clickstream data.
+
+A single journey is understood in the context of a _session_. Along the journey a _visitor_ undertakes a number of _user actions_.
+
+Clickstream analytics concerns questions like:
```

Comment:
I'd move this description to the very top, above Background. I had to read through quite a bit of content before finding out what clickstream analytics are.